### ISSUE TITLE
Consolidate String repos

### ISSUE THREAD

**Issue by lorenzo** [elm-community/Manifesto#35]
Today in the slack channel there was a discussion about the multiple `String` related repos. It would be nice to consolidate the String extras in a single repository. I'm willing to take that job if you think it is a good idea.

**Comment by jaapz** [comment]
Yeah, I'd propose looking at the API for [underscore.string](https://epeli.github.io/underscore.string/) and implement the ones that are not covered by the String api already.

I think having a kind of canonical String-extra's repo would be nice to have, without having to bloat the standard library's String with too many helper functions.

I'm interested in helping out to get this working.

**Comment by mgold** [comment]
I've created a repo [here](https://github.com/elm-community/string-extra) and given you both write access. Have at it!

(Can this be closed?)

**Comment by jvoigtlaender** [comment]
I guess: close only after https://github.com/elm-community/Manifesto/blob/master/maintainers.md was updated?

**Comment by mgold** [comment]
Sounds reasonable. By the way @lorenzo @jaapz would you mind letting some folks know when you want to release 1.0.0 and they can do code review? Just @-tag whoever or post to slack, the mailing list, etc.

**Comment by lorenzo** [comment]
Will do :)

**Comment by eeue56** [comment]
Can this be closed?
